tests/js/jest.config.js

```diff
@@ -22,11 +22,25 @@ import { fileURLToPath } from 'url';
 
 const __dirname = dirname(fileURLToPath(import.meta.url));
 
+/** @type {import('@jest/types').Config} */
 export default {
   rootDir: '../../',
   resolver: '@web-stories-wp/jest-resolver',
+  preset: 'ts-jest',
   transform: {
-    '^.+\\.[jt]sx?$': 'babel-jest',
+    '^.+\\.tsx?$': [
+      'ts-jest',
+      {
+        // Disable type checking but improve performance.
+        // See https://kulshekhar.github.io/ts-jest/docs/getting-started/options/isolatedModules
+        // See https://github.com/kulshekhar/ts-jest/issues/1648
+        // TODO: Look into disabling this again.
+        isolatedModules: true,
```

Comment:
Not really happy about this but otherwise tests won't work at all since it fails to find some types.